great_expectations/validator/validator.py

```diff
@@ -312,7 +312,7 @@ def inst_expectation(*args, **kwargs):
 
             if self.interactive_evaluation:
                 configuration.process_evaluation_parameters(
-                    self._expectation_suite.evaluation_parameters,
+                    self.expectation_suite.evaluation_parameters,
```

Comment:
if expectation suite has been "private" all this time, do we want to expose it now?

Reply:
you're right! I want to expose a read-only value through the `@property` but I think internal usage should remain consistent. Changing with this next commit.